cmd/admin-policy-list.go

```diff
@@ -42,24 +42,24 @@ FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
 EXAMPLES:
-  1. List all policies.
+  1. List all policy.
```

Comment:
Inconsistent with `Usage: list all policies`
Either `List all policy` should be in plural or `Usage:` needs to be singular.